code/game/gamemodes/miniantags/abduction/machinery/console.dm

```diff
@@ -29,7 +29,13 @@
 	var/list/datum/icon_snapshot/disguises = list()
 
 /obj/machinery/abductor/console/Initialize()
-	..()
+	. = ..()
```

Comment:
But `SSatoms` is using the return value of `Initialize`? I understand that discarding the default `INITIALIZE_HINT_NORMAL` and implicitly returning `null` yields the same result (they are both zero), but that feels like bad form.

I fully agree doing so on `ComponentInitialize` / `LateInitialize`, just not on `Initialize` itself. Unless I'm missing something?

Reply:
Ah okay my initial review was correct. It is because you return another value right afterwards. Doing late night reviews ain't always the best idea 😅

Reply:
Oops! Damn you github code reviews!